deployments/helm-chart/README.md

```diff
@@ -159,6 +159,7 @@ Parameter | Description | Default
 `controller.image.tag` | The tag of the Ingress Controller image. | 2.3.0
 `controller.image.pullPolicy` | The pull policy for the Ingress Controller image. | IfNotPresent
 `controller.config.name` | The name of the ConfigMap used by the Ingress Controller. | Autogenerated
+`controller.config.customConfigMap` | The name of the custom ConfigMap used by the Ingress Controller. | ""
```

Comment:
I think I would have it as a separate field, not as part of `config`. Something like `controller.customConfigMap`.